packages/cli/src/commands/remove.ts

```diff
@@ -167,9 +169,15 @@ export default async function main(client: Client) {
         })
       );
 
-      projectDeployments
-        .slice(0, 201)
-        .map(pDeployments => deployments.push(...pDeployments));
+      // only process the first 201 projects
```

Comment:
**question (non-blocking):** why this limit?

This comment says what is happening, but I'm more interested in why. Is this a limit of the system somewhere?

Reply:
I have no idea. I was curious about that limit also, but I have no idea hence the comment. I'm happy to remove the `.slice()` if there's no reason for the limit.